iam/policy-templates/dss-checkout-sfn-lambda.json

```diff
@@ -12,18 +12,11 @@
       ],
       "Resource": "arn:aws:logs:*:*:*"
     },
-    {
```

Comment:
this should be removed, right?

Reply:
@ttung please explain

Reply:
Hahaha, I'm the worst.  You are right, I am wrong, carry on. :)